lib/core/src/Cardano/Wallet/Network.hs

```diff
@@ -248,6 +251,26 @@ defaultRetryPolicy =
 
 type family GetStakeDistribution target (m :: * -> *) :: *
 
+-- | Use the HFC history interpreter to get the slot and epoch lengths current
+-- for the network tip.
+--
+-- This may throw a 'PastHorizonException' in some cases.
+getSlottingParametersForTip
+    :: Monad m
+    => NetworkLayer m target block
+    -> m SlottingParameters
+getSlottingParametersForTip nl = do
+    tip <- either (const 0) slotNo <$> runExceptT (currentNodeTip nl)
+
+    -- TODO: Query activeSlotCoeff. Where to get it from though? Need to be able
+    -- to query Globals from ledger.
+    let getActiveSlotCoeff = pure (ActiveSlotCoefficient 1.0)
```

Comment:
We can't really hard-code this value because we use a different value for testing than what's use on the mainnet and testnet networks :/ 
I'd suggest to keep the active slot coefficient untouched and keep retrieving it from the Shelley genesis. The value for Byron is always known and equal to `1.0` so we could have this function return `1.0` or the shelley value depending on the era we're in?

In the meantime, let's ask if the `Globals` can be made available through the LSQ protocol.

Reply:
> In the meantime, let's ask if the Globals can be made available through the LSQ protocol.

Think Rodney already asked -> https://input-output-rnd.slack.com/archives/CFKLUH4R0/p1603176743124800, and it looked like they were going to add it.

Reply:
Yes, I could add an era switch for hardcoding the mainnet activeSlotCoeff value, or we could wait for it to be implemented in shelley-ledger and cardano-node.